.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.vscode
```

Comment:
This should be in a global `.gitignore` really (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).  I prefer to keep only repository specific files in the respository `.gitignore`.